src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$color_1: #616070;
+$color_2: rgba(6, 11, 53, 1);
+$color_3: white;
+$color_5: rgba(97, 96, 112, 1);
+$font-family_1: "Roboto", sans-serif;
+$light-blue: rgba(0, 172, 220, 1);
+$background-color_2: white;
```

Comment:
Rename your variables to give them some meaning e.g.
`color-primary` `color-secondary` `color-background` `font-family-main` and so on